### ISSUE TITLE
Cross-repository projects

### ISSUE THREAD

**Issue by phlax** [dear-github/dear-github#209]
quite often a project for us will include related issues/prs/etc from several repositories.

It would be incredibly useful if you were able to reference entities from other repositories in a project.

**Comment by koenpunt** [comment]
Just use `user/repo#123` to refer to issues and/or pull request in other repos

**Comment by phlax** [comment]
hi @koenpunt - sorry i should have been more specific - i was hoping to be able to reference prs/tickets with cards in a project - title changed.

**Comment by koenpunt** [comment]
In that case you can create a project under an organization: https://github.com/orgs/NAME/projects

**Comment by phlax** [comment]
brilliant @koenpunt that mostly resolves the issue, thanks.

the only other case is that we would like to be able to reference upstream tickets etc from a private repo - ie we are working on internal jobs or addressing client issues that depend on upstream tickets

**Comment by waehlisch** [comment]
Actually, what we need is to create cards referring to issues/PRs across organizations. The general search interface (https://github.com/search) allows searching among repos of all users/organizations. However, the search interface in "Add cards" is limited to the current organization. Would be great to change this.

**Comment by epoberezkin** [comment]
> @koenpunt In that case you can create a project under an organization

A few problems with that approach
- there is no way to move the existing project from repo to org
- it often makes more sense to keep it in the main project repo, rather than in the org
- as @waehlisch wrote some companies have multiple orgs

**Comment by tomalec** [comment]
:+1: for 
> Actually, what we need is to create cards referring to issues/PRs across organizations.

Creating a card with just a link to external org/repo/issue does not entirely solve the problem, as 
- the status of the external issue is not reflected as for local ones.

**Comment by rickmanelius** [comment]
+1 for this. Sometimes there is a very specific reason to have multiple organizations and being able to track across would be helpful.

**Comment by epoberezkin** [comment]
seems done

**Comment by mroderick** [comment]
> seems done

I haven't been able to find anything that suggests this is done. If you know of anything that shows it is done, could you post a url to it?

**Comment by katemihalikova** [comment]
Having the same problem, trying it now reveals that when I press ➕ button at a column _(Add a note to this column)_ and then put the long issue identificator into the textarea with _Enter a node_, it correctly loads the issue from another repo/user/org. Just press _Add card_ to add it.

![image](https://user-images.githubusercontent.com/7225023/34945884-1bb9b208-fa05-11e7-84a7-ba3b49a17642.png)

**Comment by tomalec** [comment]
Cool, almost there. But it will still leave no trace, nor reference in the issue itself.

**Comment by mrhillsman** [comment]
just buy ZenHub ( zenhub.com ) and have their code/resources and people work to take projects in GitHub to infinity and beyond 😄

**Comment by arvenil** [comment]
Indeed @koenpunt  suggestion to create org wide project is great - I didn't spot that earlier, where it is hiding? It's great, but still not enough, the best would be to be able create personal/independent projects, for all organizations and repositories. Currently I can create project per organization, but I can't create it for just my user (I need to create organization).

**Comment by svnpenn** [comment]
https://github.com/dear-github/dear-github/issues/93

**Comment by svnpenn** [comment]
You can do this with projects:

1. Click "Add a note to this column"
2. Paste link to issue from another repo
3. Click "Add"

http://help.github.com/articles/about-project-boards

Alternatively if you create an organization you can add cards for issues from
any repo owned by the org.

- http://help.github.com/articles/creating-a-new-organization-from-scratch
- http://help.github.com/articles/managing-project-boards-in-your-repository-or-organization

**Comment by adrinjalali** [comment]
>  @koenpunt In that case you can create a project under an organization

This doesn't cover the usecase where the individual is not a member of the organization to which they're contributing, which happens very often when contributing to open source projects.

**Comment by adamwdennis** [comment]
Zenhub works really well for this.

**Comment by roborourke** [comment]
What's the reason for the 5 linked repo limit?

**Comment by louisl** [comment]
> What's the reason for the 5 linked repo limit?

Agreed, could do with more...

One of my projects is like this.

- api-spec-repo
- api-src-repo
- angular-api-client-repo
- ios-api-client-repo
- angular-front-end-repo

Oh no, hit the limit!
- ios-front-end-repo
- data-conversion-repo

**Comment by rosibaj** [comment]
Seconding the need for:
- more than 5 linked repos on a organization board
- option for a CROSS-ORGANIZATION board that uses the actual tickets that tracks actual ticket progress, not just a note format

**Comment by jay-hawkins** [comment]
third on this - I don't get the 5 repo limit. I'd like to be able to use a project page as a kanban board for a team we currently have 20+ repos that we actively develop

**Comment by lukehefson** [comment]
Hello folks! 👋 I know this isn't quite solving for the problem in the orginal post above ☝️ (and we're definitely investigating that!) but I thought I'd let you know that we extended the limit on the number of repos you can link to a board to 25: https://github.blog/changelog/2019-03-20-increasing-the-number-of-repositories-that-can-be-linked-to-a-project-board/

We would have gone higher, but for now that's the best we can do in regards to performance. We'll be iterating as ever though! Keep the feedback coming!

**Comment by rosibaj** [comment]
25 solves a lot of problems!  Thanks for the update @lukehefson!

**Comment by louisl** [comment]
Nice one @lukehefson, thanks!

**Comment by jay-hawkins** [comment]
@lukehefson this is great news! thanks for listening as always

**Comment by pranereddy** [comment]
@lukehefson Great, when will this feature be available on ghe?

**Comment by lukehefson** [comment]
@pranereddy I don’t have an exact date, but it’ll be in the next major update (2.17) 🙌

**Comment by hdevalence** [comment]
Lots of open-source work requires shepherding some changes into various upstream repositories, but because project cards are siloed into repositories or organizations, it isn't possible to track this work.   It would be really great if project cards could be created from *any* repository, so that people or orgs can keep track of all of the upstream changes they need to get done.  

This would also enable entirely new Github-centered workflows -- for instance, someone could create a project board for "all of their open-source work", acting as a personal view into issues on various other repositories.

**Comment by andrechr** [comment]
@lukehefson thx for the update man. 
But still in automate kanban the note is not moved
how do we make it automate?

**Comment by lukehefson** [comment]
> lukehefson thx for the update man.
> But still in automate kanban the note is not moved
> how do we make it automate?

Hi @andrechr! 👋 Sorry, but I'm not sure I fully understand what you mean 😕. Could you send a bug report over to support@github.com (and feel free to mention my name if you'd like) and we'll dig in!

**Comment by jnystrom** [comment]
But what about creating a project card that is cross repos?  We have a ton of repos, and an item could deal with multiple repos.  Today, I would have to create a card for each repo that is part of the task, right?  No way to group them all into a single task, right?

**Comment by zmackie** [comment]
We're trying to solve the "cross organization" issue problem for another reason: I'm the PM for a group of people from one company that contributes upstream to an OSS project. I want to manage *our* backlog as a subset of the larger project, but with different priorities and more granularity. I'd love to use a project here, but it can't live in the main org. Thus, I lose lots of the automation and UX niceties. Plus 💯 for cross-org projects. BTW anecdotally, I talked to three other PMs in my company that have this problem and all of them said the lack of first-class support for cross-org was a major pain point.

**Comment by epignatelli** [comment]
Also it may be extremely useful if a card could co-exist in two different organisation-level projects.
For example, you might use a project to organise the cards cronologically and another to organise them by the area of development.
This is now possible for `issue`s, `pr`s and `project`s, but not for notes.

**Comment by mvincerx** [comment]
> Having the same problem, trying it now reveals that when I press ➕ button at a column _(Add a note to this column)_ and then put the long issue identificator into the textarea with _Enter a node_, it correctly loads the issue from another repo/user/org. Just press _Add card_ to add it.
> 
> ![image](https://user-images.githubusercontent.com/7225023/34945884-1bb9b208-fa05-11e7-84a7-ba3b49a17642.png)

Thanks, @katemihalikova! Your solution worked for me.

**Comment by mmilenkovic** [comment]
@mvincerx Yes, it loads the issue indeed but the problem for me is that the status of the issue is not propagated and shared between the projects.
<img width="353" alt="Screenshot 2020-04-16 at 12 37 12" src="https://user-images.githubusercontent.com/2448210/79446875-4331e780-7fdf-11ea-8119-6345901e667e.png">
I would like to have an option to view one issue in multiple projects/boards, so that once you update the issue in one project/board the update is propagated to all. Given the example above, if the issue is moved to done in one, it's moved to done in the other one as well.

**Comment by PolarBean** [comment]
I'm working with the same issue as mmilenkovic, is there a way to propogate status for cross project cards?
